src/base/setup.md

```diff
@@ -1,50 +1,140 @@
-# Prerequisites
-If you do not have `substrate` installed on your machine, run:
+# Installation and Running Recipe Kitchen Node
+
+## Setup
+
+If you do not have Subtrate development environment setup on your machine, please install it as followed.
+
+### For Linux / macOS
 
 ```bash
+# Setup Rust and Substrate
 curl https://getsubstrate.io -sSf | bash
 ```
 
-While the code compiles, read about how the [Substrate runtime architecture](https://substrate.dev/docs/en/runtime/architecture-of-a-runtime) composes [modules](https://substrate.dev/docs/en/runtime/substrate-runtime-module-library) to configure a runtime. 
+### For Windows
+
+Refer to our [Substrate Installation on Windows](https://substrate.dev/docs/en/next/getting-started#getting-started-on-windows).
 
-## Module
+## Running the Recipe Kitchen Node
 
-*At the moment*, this resource focuses primarily on module development patterns, though there are plans to add examples of interesting runtime configurations using the existing modules. To develop in the context of the module, it is sufficient to clone the [module-template](https://github.com/substrate-developer-hub/substrate-module-template)
+To interact with the code in this recipe, `git clone` the source repository. We also want to kick-start the node compilation as it may take about 30 minutes to complete depending on your hardware.
```

Comment:
```diff
- To interact with the code in this recipe, `git clone` the source repository. We also want to kick-start the node compilation as it may take about 30 minutes to complete depending on your hardware.
+ To interact with the code in these recipes, `git clone` the source repository. We also want to kick-start the node compilation as it may take about 30 minutes to complete depending on your hardware.
```